auto_scan.py

```diff
@@ -106,6 +106,12 @@ def get_active_containers(lw_client, container_registry_domains, start_time, end
 
     return active_containers
 
+def list_containers(containers):
+    for container in containers:
+        # Parse the container registry and repository
+        container_registry, container_repository = container['REPO'].split('/', 1)
+        print(f'{container_registry}/{container_repository}:{container["TAG"]}')
```

Comment:
Can this be simplified?

```suggestion

def list_containers(containers):
    for container in containers:
        print(f'{container["REPO"]}:{container["TAG"]}')
```

Reply:
Good catch! 🤦